packages/sdk-metrics/package.json

```diff
@@ -83,7 +83,7 @@
     "webpack-merge": "5.10.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.3.0 <1.10.0"
+    "@opentelemetry/api": "^1.3.0"
```

Comment:
This is on purpose. It will break every time a new feature is added to the API.
The API is backward compatible for users, but not for implementors. AFAIK it is like this in every OpenTelemetry Langauge implementation.

See the spec for details:
https://github.com/open-telemetry/opentelemetry-specification/blob/a03382ada8afa9415266a84dafac0510ec8c160f/specification/upgrading.md?plain=1#L97-L122